src/data_handler/inc/data_handler.h

```diff
@@ -49,7 +49,10 @@ extern "C"
 
     SHARED_EXPORT int CALLING_CONVENTION get_avg_band_powers (double *raw_data, int rows, int cols,
         int sampling_rate, int aply_filters, double *avg_band_powers, double *stddev_band_powers);
-
+    // logging methods
+    SHARED_EXPORT int CALLING_CONVENTION set_log_level (int log_level);
+    SHARED_EXPORT int CALLING_CONVENTION set_log_file (char *log_file);
+    SHARED_EXPORT int CALLING_CONVENTION log_message (int log_level, char *message);
```

Comment:
I think this method should be removed and seems like there is no implementation for it in cpp file. Probably you forgot to remove it from header

Reply:
You were referring only to the log_message method right.

Reply:
yes